install/google_cloud_sql/_01_create_monitoring_user_psql.mdx

```diff
@@ -0,0 +1,18 @@
+import MonitoringUserSetupInstructions from "../../components/MonitoringUserSetupInstructions";
+import VerifyConnection from "../../components/VerifyConnection";
+
+Connect to the database you will be monitoring, again as a user with **cloudsqlsuperuser / alloydbsuperuser** privileges.
+Then run the following to create a monitoring user (we've generated a random password for you, but you
+can replace it with one of your choosing):
+
+<MonitoringUserSetupInstructions />
+
+Note that it is important you run these as a user with **cloudsqlsuperuser / alloydbsuperuser** privileges in order to pass down the full access to statistics tables.
+
+## Verify connection
+
+Let's make sure we can connect to the database with the new user:
+
+<VerifyConnection host="1.2.3.4" />
+
+Write down the username and password of the monitoring user, we will use it in the last step of this tutorial.
```

Comment:
```suggestion
Write down the username and password of the monitoring user, we will use it in the last step of this guide.
```

Reply:
I was wondering about this word "tutorial" as I don't really remember that I was writing down this.
Turned out that we actually use this sentence in the other places, and I copy & pasted from there, that's why it's there.
I'll go ahead and update places that say "tutorial" too.